### ISSUE TITLE
Resolving OOM Issues in train_control.py on 4 A100 GPUs (80GB Each)

### ISSUE THREAD

**Issue by Beijia11** [aigc-apps/CogVideoX-Fun#102]
My settings are as follows. Is there any way to reduce OOM (Out of Memory) issues?
CUDA_VISIBLE_DEVICES=0,1,2,3 accelerate launch --main_process_ip=10.140.37.162 --main_process_port=12345 --num_machines=1 --num_processes=4 --machine_rank=0  scripts/train_control.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --train_data_dir=$DATASET_NAME \
  --train_data_meta=$DATASET_META_NAME \
  --image_sample_size=1024 \
  --video_sample_size=256 \
  --token_sample_size=512 \
  --video_sample_stride=3 \
  --video_sample_n_frames=49 \
  --train_batch_size=1 \
  --video_repeat=1 \
  --gradient_accumulation_steps=4 \
  --dataloader_num_workers=1 \
  --num_train_epochs=100 \
  --checkpointing_steps=50 \
  --learning_rate=2e-05 \
  --lr_scheduler="constant_with_warmup" \
  --lr_warmup_steps=50 \
  --seed=43 \
  --output_dir="output_dir" \
  --gradient_checkpointing \
  --mixed_precision="bf16" \
  --adam_weight_decay=3e-2 \
  --adam_epsilon=1e-10 \
  --vae_mini_batch=1 \
  --max_grad_norm=0.05 \
  --random_hw_adapt \
  --training_with_video_token_length \
  --enable_bucket \
  --trainable_modules "."

**Comment by LuminousLark** [comment]
I'm also encountering the same OOM (Out of Memory) issue. Are there any good solutions? I'm using 8 A100 GPUs.
